lib/app/sequence_base.rb

```diff
@@ -712,6 +712,24 @@ def validate_reference_resolutions(resource)
         assert(problems.empty?, problems.join("<br/>\n"))
       end
 
+      def save_delayed_sequence_references(resource)
+        non_patient_sequences = ['Location', 'Organization', 'Medication', 'Practitioner', 'PractitionerRole']
```

Comment:
We really should avoid globally defining resources for special treatment. This is especially problematic since this list exists here and in `TestingInstance`. Rather than maintaining this list, I think each non-patient sequence should declare itself to be one.

Reply:
made it so sequences mark themselves as delayed